pkg/operator/starter.go

```diff
@@ -42,7 +42,10 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
-	dynamicClient, err := dynamic.NewForConfig(ctx.KubeConfig)
+	dynamicClientConfig := dynamic.ConfigFor(ctx.KubeConfig)
+	dynamicClientConfig.Burst = 40
```

Comment:
we need to experiment and find good numbers for `QPS/Burst`

Reply:
Please rename to `migrationClientConfig` and verify that it is only used for migration

Reply:
Are those the numbers upstream migrator uses as well?